packages/components/cascader-panel/src/types.ts

```diff
@@ -21,10 +21,7 @@ export type RenderLabel = ({
   data: CascaderOption,
 }) => VNode | VNode[]
 
-export enum ExpandTrigger {
-  CLICK = 'click',
-  HOVER = 'hover',
-}
+export type ExpandTrigger = 'click' | 'hover'
```

Comment:
We can import `ExpandTrigger`, `CascaderOption` and `CascaderProps` from `node.ts`

```ts
import type {
  default as CascaderNode,
  CascaderOption,
  CascaderProps,
  ExpandTrigger,
} from './node'

export type { CascaderNode, CascaderOption, CascaderProps, ExpandTrigger }
```